modules/client/examples/01-client/12-prepare-uninstallation.ts

```diff
@@ -0,0 +1,94 @@
+/* MARKDOWN
+---
+title: PrepareInstallation
+---
+
+### Prepare Uninstallation of a plugin
+
+*/
+
+import {
+  Client,
+  GasFeeEstimation,
+  PrepareUninstallationParams,
+  PrepareUninstallationSteps,
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiate the general purpose client from the Aragon OSx SDK context.
+const client: Client = new Client(context);
+
+const prepareUninstallationParams: PrepareUninstallationParams = {
+  daoAddressOrEns: "0x1234567890123456789012345678901234567890", // my-dao.dao.eth
+  pluginAddress: "0x2345678901234567890123456789012345678901",
+  uninstallationParams: [ // Parameters needed by the prepare uninstall abi
+    "test",
+    ["testArrayItem", "testArrayItem2"],
+    {
+      testObjectProp: "testObjectValue",
+    },
+  ],
+  uninastallationAbi: ["uint256", "string", "string[]", "object"], // optional, if not specified, the abi will be fetched from the plugin metadata
```

Comment:
The ABI is defined in terms of Solidity, there is no such data type called `object`. It should rather be a `tuple()` but this is being defined right now. So I would probably remove the last parameter and just leave a number, and an address.

To another note, the format that we will probably use is not Ether's but rather the standard solidity one (a bit more verbose)